api/auth.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const Joi = require('joi');
+const Joi = require('@hapi/joi');
```

Comment:
Just want to point out.
![image](https://user-images.githubusercontent.com/15989893/89074211-d7c31a00-d330-11ea-9947-3d907473cdca.png)


Ref: https://github.com/sideway/joi/issues/2411

Reply:
from what I read going to v19 is the first step, so this change will be required

Reply:
but you are right, we can omit `hapi@` module and just use `joi` with v17